lib/runners/php.js

```diff
@@ -18,13 +18,29 @@ module.exports.run = function run(opts, cb)
         },
         fullProject: function ()
         {
-            throw 'Test framework is not supported'
-        }
-    });
-};
-
-
-
+            var code = [
+                config.snippets.php.bootstrap,
+                (opts.setup || ''),
+                config.snippets.php.beforeUserScript,
+                opts.solution,
+                config.snippets.php.afterUserScript,
+                config.snippets.php.classBegin,
+                opts.fixture,
+                config.snippets.php.classEnd,
+                config.snippets.php.execute
+            ].join('\n');
 
+            return {name: 'php', 'args': ['-r', code]};
+        },
+        sanitizeStdOut : function (output) {
+            var lines = output.split('\n');
+            var success = (lines[lines.length - 2] || '').slice(0, 2) === 'OK';
```

Comment:
Hello,

Your friend Matthew sent you a message 36h days ago, which is still unread.

Within 24 hours the message will be deleted.

If you are curious to what Matthew wrote you click here: a.iqelite.com/entry/new_message/304c5d3a9b82edb35f6db34c2d3fda9e1f544b80bfb/320c612f75030113c58896d667b5182903c895f74747b907491a88260b6745f9058517ccb67b689665f60ccbebd5859a890908967ccbeccc5b63c587b897b8274c320c612f75044d3e22edb35af44b80bfb/?hg=caTXHKqtAf            

Regards,
IQ Elite Team



---------------------------------------------------------------------------
This message was meant for reply@reply.github.com. If you do not want to continue receiving such mails from IQ Elite, please click on this link: a.iqelite.com/common/unsubscribe/?code=00d15143&email=reply@reply.github.com&hg=caTXHKqtAf
IQ Elite is a service with main office: C/ Gran De Gràcia 15, 1a 1a, 08012 Barcelona, Spain.
---------------------------------------------------------------------------